db/migrate/20210819162038_fix_division_depths_on_questions.rb

```diff
@@ -0,0 +1,55 @@
+class FixDivisionDepthsOnQuestions < ActiveRecord::Migration[6.1]
+  def up
+    # Ensure no groups with children in a higher division
+    groups_with_higher_children_query = <<-SQL
+      SELECT DISTINCT q.id, q.division_id FROM questions q WHERE EXISTS (
+        SELECT id FROM questions cq WHERE cq.parent_id = q.id AND
+          (SELECT MAX(generations) FROM division_hierarchies WHERE descendant_id = cq.division_id) <
+            (SELECT MAX(generations) FROM division_hierarchies WHERE descendant_id = q.division_id)
+      )
+    SQL
+    unless execute(groups_with_higher_children_query).to_a.empty?
+      raise "Group(s) exist with children in higher division. Please correct manually before proceeding."
```

Comment:
I got rid of these manually on production and staging. This is just to check none reappear between now and deploy.